TKGame/TKGame.cs

```diff
@@ -199,8 +199,9 @@ protected override void Draw(GameTime gameTime)
             // Deferred rendering means things are drawn in the ordered they're called
             spriteBatch.Begin(SpriteSortMode.Deferred, BlendState.NonPremultiplied);
 
-            //Draws the image into the Background
-            spriteBatch.Draw(Art.BackgroundTexture, levelComponent.GetCurrentStage().Background.BackgroundRect, Color.White);
+            //Draw Stage Backgrounds
+            levelComponent.GetCurrentStage().Draw(spriteBatch);
+
```

Comment:
I don't see any change to the LevelEditor.SaveJSON or LoadJSON functions (abbrev func) so currently we have to hard code the background when we instantiate a stage. ALL stage data from walls, entities, background, doors/triggers, items, etc. This is something we NEED to be implementing when we add features to the stage otherwise the refactor backlog for the next 2 sprints are gonna be hell.

Reply:
I know you have other classes/work but +50 lines of code given how simplistic they code itself that was added is not nearly good enough for the final few sprints for OUR junior project.
These next couple of sprints I NEED all of us to contribute to the best of our ability to get the game looking better and actually have some playability and to do that we need more effort going into each sprint than this.